gdnative-sys/build.rs

```diff
@@ -141,8 +141,8 @@ mod header_binding {
                         .unwrap();
 
                     println!(
-                        "Automatically chosen version : {} (latest)",
-                        ndk_version.to_string()
+                        "cargo:warning=Automatically chosen version : {} (latest)",
```

Comment:
Maybe mention the reason why this decision was made?
```rs
"cargo:warning=Multiple Android NDKs found; chosing latest version: {}"
```

(also, no space before colon)

Reply:
It's already specified in the full output :
```
warning: Multiple android ndk versions have been detected.
warning: You should chose one using ANDROID_NDK_VERSION environment variable to have reproducible builds.
warning: Available versions : ["/usr/local/lib/android/sdk/ndk/23.1.7779620", "/usr/local/lib/android/sdk/ndk/22.1.7171670", "/usr/local/lib/android/sdk/ndk/21.4.7075529"]
warning: Automatically chosen version : 23.1.7779620 (latest)
```

Sorry for the spaces before colons, it's because of French typography rules that wants non-breaking spaces before certain punctuation and I'm used to it.

Reply:
Ah, missed that. Haha, no worries, happens a lot 🙂